res/database/init.sql

```diff
@@ -1,3 +1,10 @@
+CREATE DATABASE  IF NOT EXISTS `bugcache` /*!40100 DEFAULT CHARACTER SET utf8mb4 */;
+USE `bugcache`;
+-- MySQL dump 10.13  Distrib 5.6.28, for debian-linux-gnu (x86_64)
+--
+-- Host: localhost    Database: bugcache
+-- ------------------------------------------------------
+-- Server version	5.7.11
```

Comment:
I used Workbench directly. We should anyway write a script that dumps it, if possible based on the configuration file.

That's another reason to separate configuration and Aerys host bootstrap, so we can easily include it in maintenance scripts.